### PR TITLE
feat(Mobile-Dart-Knights): add profile-page 

### PR DESCRIPTION
**Pull request by hayattofik-01**
I have finished the UI implementation for the page below. It includes all the issues on jira that are assigned to me (upper card,blue row inside the upper card,and post display section).
 
Screenshot_20230501-165514

### CODE REVIEW

File: Mobile/starter_project_dark_knights/lib/features/user_profile/presentation/screen/profile_page.dart [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+import '../widgets/post_display.dart';
+
+
+
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
```
Comment (Redi22) [inline Mobile/starter_project_dark_knights/lib/features/user_profile/presentation/screen/profile_page.dart:15]: This should only be a part of your main.dart there should only be one MaterialApp.

File: Mobile/starter_project_dark_knights/lib/features/user_profile/presentation/widgets/blueCard.dart [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+
+import '../../../../core/utils/colors.dart';
+
+
+class BlueCard extends StatelessWidget {
```
Comment (Redi22) [inline Mobile/starter_project_dark_knights/lib/features/user_profile/presentation/widgets/blueCard.dart:6]: rename to StatusCard

**Review by Redi22 — CHANGES_REQUESTED** [review]
Remove the .dart_tool folder from all the projects as well

File: Mobile/starter_project_dark_knights/lib/features/user_profile/presentation/screen/profile_page.dart (hunk 1 above)
Comment (hayattofik-01) [reply Mobile/starter_project_dark_knights/lib/features/user_profile/presentation/screen/profile_page.dart]: Done

File: Mobile/starter_project_dark_knights/lib/features/user_profile/presentation/widgets/blueCard.dart (hunk 2 above)
Comment (hayattofik-01) [reply Mobile/starter_project_dark_knights/lib/features/user_profile/presentation/widgets/blueCard.dart]: Done

**Review by Redi22 — APPROVED** [review]
LGTM

**Pull request merged**